packages/client/src/api/api.service.ts

```diff
@@ -1,4 +1,4 @@
-import {
+import type {
```

Comment:
use `import type` everywhere in the `@novu/client` so that `@novu/shared` won't be included in the bundle

Reply:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** If we are just using the types, maybe we can move the `@novu/shared` under the `devDependency` as well.‏

Reply:
We can't because of the types generated for ESM, they are not "copied" during the build time.